decred/build.py

```diff
@@ -0,0 +1,17 @@
+"""
+Copyright (c) 2020, The Decred developers
+See LICENSE for details
+"""
+
+# fmt: off
+try:
+    from Cython.Build import cythonize
+except ImportError:
+    def build(setup_kwargs):
+        pass
+else:
+    def build(setup_kwargs):
+        setup_kwargs.update(
+            {"ext_modules": cythonize(["decred/crypto/secp256k1/field.py"])}
+        )
```

Comment:
I disagree strongly. This is the very first time I hear the `else` clause of the `try` being described as "weird". The utility lies in keeping the `try` block as small as possible. From [the docs](https://docs.python.org/3/tutorial/errors.html#handling-exceptions):

"The use of the else clause is better than adding additional code to the try clause because it avoids accidentally catching an exception that wasn’t raised by the code being protected by the try … except statement."